custom_components/tahoma/button.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Overkiz number devices."""
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENTITY_CATEGORY_DIAGNOSTIC
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import OverkizDescriptiveEntity
+
+BUTTON_DESCRIPTIONS = [
+    # My Position (cover, light)
+    ButtonEntityDescription(
+        key="my",
+        name="My Position",
+        icon="mdi:star",
+    ),
+    # Wink (light)
+    ButtonEntityDescription(
+        key="wink",
+        name="Wink",
+        icon="mdi:emoticon-wink",
+    ),
+    # Identify
+    ButtonEntityDescription(
```

Comment:
What do you think to add start and stop identify as disable by default?

Reply:
Let's test identify first, if this does the job, I am not sure if we need start / stop identify as well.

Reply:
@tetienne could you try this? I am not close to my tahoma covers/lights for a while...

Reply:
```
/config/custom_components/tahoma/button.py:73: RuntimeWarning: coroutine 'OverkizExecutor.async_execute_command' was never awaited
Coroutine created at (most recent call last)
  File "/usr/local/bin/hass", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/homeassistant/__main__.py", line 318, in main
    exit_code = runner.run(runtime_conf)
  File "/usr/local/lib/python3.9/site-packages/homeassistant/runner.py", line 121, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 629, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 596, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1882, in _run_once
    handle._run()
  File "/usr/local/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/local/lib/python3.9/site-packages/homeassistant/components/button/__init__.py", line 114, in _async_press_action
    await self.async_press()
  File "/config/custom_components/tahoma/button.py", line 73, in async_press
    self.executor.async_execute_command(self.entity_description.key)
  self.executor.async_execute_command(self.entity_description.key)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback

```